### PR TITLE
Add owner e-mail and job type labels for consumption by the cpu prediction model

### PR DESCRIPTION
**Pull request by gabrielhartmann**
### Description of the Change

These should be the last attributes needed by the CPU prediction model for some time.

### CODE REVIEW

File: executor/runtime/container.go [hunk 1]
```diff
@@ -45,6 +49,8 @@ func NewContainer(taskID string, titusInfo *titus.ContainerInfo, resources *runt
 
 	env := cfg.GetNetflixEnvForTask(titusInfo, strMem, strCPU, strDisk, strNetwork)
 	labels[appNameLabelKey] = titusInfo.GetAppName()
+	labels[ownerEmailLabelKey] = titusInfo.PassthroughAttributes[ownerPassThroughKey]
+	labels[jobTypeLabelKey] = titusInfo.PassthroughAttributes[jobTypePassThroughKey]
```
Comment (gabrielhartmann) [inline executor/runtime/container.go]: It's my understanding that if a value isn't present in the map you get the `zero value` which for strings is the empty string, which for me is desired behavior here.

File: executor/runtime/container.go [hunk 2]
```diff
@@ -51,6 +55,12 @@ func NewContainer(taskID string, titusInfo *titus.ContainerInfo, resources *runt
 	labels[diskLabelKey] = strDisk
 	labels[networkLabelKey] = strNetwork
 
+	passthroughAttributes := titusInfo.GetPassthroughAttributes()
+	if passthroughAttributes != nil {
+		labels[ownerEmailLabelKey] = passthroughAttributes[ownerPassThroughKey]
+		labels[jobTypeLabelKey] = passthroughAttributes[jobTypePassThroughKey]
```
Comment (gabrielhartmann) [inline executor/runtime/container.go]: It's my understanding that if a value isn't present in the map you get the zero value which for strings is the empty string, which for me is desired behavior.

File: executor/runtime/container.go [hunk 3]
```diff
@@ -9,13 +9,17 @@ import (
...
-	workloadTypeLabelKey   = "com.netflix.titus.workload.type"
-	titusTaskInstanceIDKey = "TITUS_TASK_INSTANCE_ID"
+	appNameLabelKey          = "com.netflix.titus.appName"
+	cpuLabelKey              = "com.netflix.titus.cpu"
+	memLabelKey              = "com.netflix.titus.mem"
+	diskLabelKey             = "com.netflix.titus.disk"
+	networkLabelKey          = "com.netflix.titus.network"
+	workloadTypeLabelKey     = "com.netflix.titus.workload.type"
+	ownerEmailLabelKey       = "com.netflix.titus.owner.email"
+	ownerEmailPassThroughKey = "titus.agent.ownerEmail"
```
Comment (sargun) [inline executor/runtime/container.go]: why not `com.netflix.titus`?

Comment (gabrielhartmann) [reply executor/runtime/container.go]: This is the labeling scheme from the control plane as suggested by @corindwyer.
https://github.com/Netflix/titus-control-plane/blob/master/titus-server-master/src/main/java/com/netflix/titus/master/mesos/DefaultV3TaskInfoFactory.java#L66

**Pull request merged**